sale_report_delivered_subtotal/reports/sale_report.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleReport(models.Model):
+    _inherit = "sale.report"
+
+    price_subtotal_delivered = fields.Float(
+        string='Total Price Delivered',
+        readonly=True,
+    )
+
+    def _select(self):
+        select_str = super(SaleReport, self)._select()
+        select_str += """,
+            sum(qty_delivered * l.price_unit) as price_subtotal_delivered
```

Comment:
In case that taxes are included in the price, this won't be correct, so the correct tentative formula is:
```
sum(l.price_subtotal / l.product_uom_qty * l.qty_delivered) AS price_subtotal_delivered
```